section/storage.html

```diff
@@ -54,9 +54,25 @@ <h3>Storage</h3>
                 </dd>
 
                 <dt><dfn>maxRateOfPay</dfn></dt>
-                <dd>The maximum allowed rate of payment, expressed in user wallet asset units per hour,
-                  acting as a safety limit to prevent excessive spending.
-                  <br><span class="note">This limit is configured by the user.</span></dd>
+                <dd>
+                  The user's configured Web Monetization spending budget, expressed as a base-10 non-negative integer string in
```

Comment:
Not following the `base-10` part here - why it's relevant, and what does it exactly mean?